database/queries.js

```diff
@@ -0,0 +1,15 @@
+const pgp = require( 'pg-promise' )()
+const db = pgp({ database: 'doitnow' })
+
+const allTasks = 'SELECT * from todos'
+const createTask = 'INSERT INTO todos (id, name) VALUES ($1, $2) RETURNING *'
+
+
+const Tasks = {
+  getAll: () => {
```

Comment:
can you separate it out so it's easier to read? and less things are happening in one file? 
for example -- your queries to the database, like on lines 4 and 5, and things like that are in one file and then exported at the bottom and then used in another file.